Spanner/src/Connection/Grpc.php

```diff
@@ -120,11 +122,25 @@ class Grpc implements ConnectionInterface
      */
     private $longRunningGrpcClients;
 
+    private function enableConnectionManagement($conf_path)
```

Comment:
We use camel case variable names throughout the library, could we please stick to that convention here as well? :)

Reply:
Done.